packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -235,7 +238,7 @@ const App = (props) => {
     }, [])
 
     // Handle updating the shopper context
-    useUpdateShopperContext()
+    // useUpdateShopperContext()
```

Comment:
@sf-mkosak this should not be commented out. Please uncomment, doing so might result in new work in reconciling the functionality